spec/models/like_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
- [ ] Please use `require_relative` for local files. An error is shown with `require` since it uses the `LOAD_PATH` to locate the files.
![image](https://github.com/Obimbo07/tdn/assets/45460192/b6b46a41-df68-4409-a7bd-4b63fb5d637f)